src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -3,648 +3,656 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-# pylint: disable=too-many-lines
-from knack.help_files import helps
 
+from knack.help_files import helps
 
-helps['appservice'] = """
+helps["appservice"] = """
```

Comment:
CLI repo conventions we want to preserve:
`helps['appservice']` : single quotes
`""" docstring """` : double quotes

So the double quotes around the keys need to be single.